README.md

```diff
@@ -69,10 +69,9 @@ You can also check our [blog post](https://www.bloco.io/blog/faker-a-library-to-
  - Implement remaining components of the [Faker ruby gem](https://github.com/stympy/faker/).
 
 ## Proguard
+If you are using R8 the shrinking and obfuscation rules are included automatically.
 
-To use Faker with proguard, here are the rules you need to add:
-
-    # SnakeYAML
+ProGuard users will need to add the following options:
```

Comment:
I think something like "Otherwise, ProGuard users" is missing here. And we should either use Proguard or ProGuard consistently.